modules/programs/alacritty.nix

```diff
@@ -0,0 +1,47 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.alacritty;
+
+in
+
+{
+  options = {
+    programs.alacritty = {
+      enable = mkEnableOption "Alacritty";
+
+      settings = mkOption {
+        type = types.attrs;
```

Comment:
The `types.attrs` option does not work well with configuration merging. I'm fine with having it like this for now for simplicity sake, but eventually somebody will run into problems because of it 🙂

Reply:
I would argue we should probably wait to fix this until nixos/rfcs#42 is merged, which should resolve the problem by introducing new types.